openedx/core/djangoapps/user_authn/urls_common.py

```diff
@@ -23,37 +23,64 @@
     url(r'^create_account$', register.RegistrationView.as_view(), name='create_account'),
 
     # Moved from user_api/legacy_urls.py
+    url(
+        r'^api/user/v1/account/registration/$',
+        register.RegistrationView.as_view(),
+        name="user_api_registration"
+    ),
     # `user_api` prefix is preserved for backwards compatibility.
     url(r'^user_api/v1/account/registration/$', register.RegistrationView.as_view(),
-        name="user_api_registration"),
+        name="user_api_registration_legacy"),
 
     # V2 is created to avoid backward compatibility issue with confirm_email
+    url(
+        r'^api/user/v2/account/registration/$',
+        register.RegistrationView.as_view(),
+        name="user_api_registration_v2"
+    ),
+    # legacy url
     url(r'^user_api/v2/account/registration/$', register.RegistrationView.as_view(),
-        name="user_api_registration_v2"),
+        name="user_api_registration_v2_legacy"),
 
     # Moved from user_api/urls.py
-    # `api/user` prefix is preserved for backwards compatibility.
     url(
         r'^api/user/v1/validation/registration$',
         register.RegistrationValidationView.as_view(),
         name='registration_validation'
     ),
+    # `api/user` prefix is preserved for backwards compatibility.
+    url(
+        r'^api/user/v1/validation/registration$',
```

Comment:
This is the same URL as the previous entry.  It won't be used, right? It maps to the same view anyway, so there's no need for both of them.

Reply:
Yes, this is a redundant URL and will not be used. I have removed it.